### PR TITLE
Refs [TASK] [34900] [Handle event bottom navigation]

### PR DESCRIPTION
**Pull request by manhhung23399**
## Related Tickets
- [#Task 34900: Handle event bottom navigation](https://edu-redmine.sun-asterisk.vn/issues/34900)
## WHAT
click item bottom navigation to switch screen
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/moviews/MainActivity.kt [hunk 1]
```diff
@@ -2,12 +2,41 @@ package com.example.moviews
...
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val fragmentManager: FragmentManager = supportFragmentManager
+        fragmentManager.beginTransaction()
+            .replace(R.id.frameContainer, HomeFragment()).commit()
```
Comment (thonc-2135) [inline app/src/main/java/com/example/moviews/MainActivity.kt]: format lại nhé, mỗi lần chấm là 1 lần xuống dòng

File: app/src/main/java/com/example/moviews/MainActivity.kt [hunk 2]
```diff
@@ -2,12 +2,41 @@ package com.example.moviews
...
+                }
+                R.id.navigationSearch -> {
+                    supportFragmentManager.beginTransaction().addToBackStack(null).replace(R.id.frameContainer, SearchFragment()).commit()
+                }
+                R.id.navigationFavorites -> {
+                    supportFragmentManager.beginTransaction().addToBackStack(null).replace(R.id.frameContainer, FavoriteFragment()).commit()
+                }
+                R.id.navigationInfomation -> {
+                    supportFragmentManager.beginTransaction().addToBackStack(null).replace(R.id.frameContainer, InformationFragment()).commit()
+                }
```
Comment (thonc-2135) [inline app/src/main/java/com/example/moviews/MainActivity.kt]: đoạn này tìm cách viết gọn lại nhé, dufng extension function, vs sắp xếp lại chỗ đặt whn thì sẽ gọn hơn nữa nha

File: app/src/main/java/com/example/moviews/MainActivity.kt [hunk 3]
```diff
@@ -2,12 +2,48 @@ package com.example.moviews
...
         setContentView(R.layout.activity_main)
+        switchFragment(HomeFragment())
+        eventClickBottomNavigationView()
+    }
 
+    private fun eventClickBottomNavigationView() {
+        bottomNavigation.setOnNavigationItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.navigationHome -> {
+                    switchFragment(HomeFragment())
```
Comment (huytq-2422) [inline app/src/main/java/com/example/moviews/MainActivity.kt]:
```suggestion
                    switchFragment(HomeFragment.newInstance())
```

File: app/src/main/java/com/example/moviews/MainActivity.kt [hunk 4]
```diff
@@ -2,12 +2,48 @@ package com.example.moviews
...
+                R.id.navigationInfomation -> {
+                    switchFragment(InformationFragment())
+                }
+            }
+            true
+        }
     }
 }
+
+fun MainActivity.switchFragment(fragment: Fragment) {
```
Comment (huytq-2422) [inline app/src/main/java/com/example/moviews/MainActivity.kt]: Tạo 1 file Activity+.kt 
viết hàm này vào trong file 
```suggestion
fun FragmentActivity.switchFragment(fragment: Fragment) {
```

File: app/src/main/java/com/example/moviews/MainActivity.kt [hunk 5]
```diff
@@ -2,12 +2,48 @@ package com.example.moviews
...
+    }
 
+    private fun eventClickBottomNavigationView() {
+        bottomNavigation.setOnNavigationItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.navigationHome -> {
+                    switchFragment(HomeFragment())
+                }
+                R.id.navigationSearch -> {
+                    switchFragment(SearchFragment())
```
Comment (huytq-2422) [inline app/src/main/java/com/example/moviews/MainActivity.kt]:
```suggestion
                    switchFragment(serachFragment)
```

**Pull request merged**